Add knative-installer component to charts (#1749)
kyma-project/kyma: resources/knative-installer/README.md

# Knative installer

## Overview

This chart packs the [knative installation script](../../components/knative-installer/README.md) as a Kubernetes job.

> Installed releases of Knative components may be found in [Dockerfile](../../components/knative-installer/Dockerfile).

Reviewer: ```suggestion
For details on installed Knative components releases, see the [Dockerfile](../../components/knative-installer/Dockerfile).
```
